<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人机大战</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>841</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 215, 134);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>载入</string>
    </property>
    <addaction name="action15x15"/>
   </widget>
   <widget class="QMenu" name="menu2">
    <property name="title">
     <string>控制</string>
    </property>
    <addaction name="actionBegin"/>
    <addaction name="actionBack"/>
   </widget>
   <widget class="QMenu" name="menu3">
    <property name="title">
     <string>棋力</string>
    </property>
    <addaction name="actionLv2"/>
    <addaction name="actionLv1"/>
   </widget>
   <addaction name="menu1"/>
   <addaction name="menu2"/>
   <addaction name="menu3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action15x15">
   <property name="text">
    <string>15x15</string>
   </property>
  </action>
  <action name="action20x20">
   <property name="text">
    <string>20x20</string>
   </property>
  </action>
  <action name="actionBegin">
   <property name="text">
    <string>restart</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>back</string>
   </property>
  </action>
  <action name="actionLv5">
   <property name="text">
    <string>lv5</string>
   </property>
  </action>
  <action name="actionLv4">
   <property name="text">
    <string>lv4</string>
   </property>
  </action>
  <action name="actionLv2">
   <property name="text">
    <string>lv2</string>
   </property>
  </action>
  <action name="actionLv1">
   <property name="text">
    <string>lv1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
